scripts/dotnet.py

```diff
@@ -612,7 +613,7 @@ def get_commit_date(
         raise ValueError('.NET Commit sha was not defined.')
 
     url = None
-    urlformat = 'https://api.github.com/repos/%s/%s/commits/%s'
+    urlformat = 'https://github.com/%s/%s/commit/%s.patch'
```

Comment:
Could we add a comment with a sample URL? It's hard to review without knowing what the response looks like

Reply:
This has been updated.